dash/dash.py

```diff
@@ -217,6 +217,9 @@ def add_url(name, view_func, methods=('GET',)):
         self._cached_layout = None
         self.routes = []
 
+        # default renderer string
+        self.renderer = 'const renderer = new DashRenderer();'
```

Comment:
Should use `var` since it's not transpiled.

Reply:
`const` is already supported in all major browsers I believe.

Reply:
It may be supported in current versions, but if someone has an old version it won't work. The default should use `var` for maximum compatibility.

Reply:
I see at least one reference to us testing in IE10 https://community.plot.ly/t/browsers-compatibility/5047/2 so yeah, better drop it to `var`.